src/cascadia/TerminalApp/TabManagement.cpp

```diff
@@ -391,6 +404,45 @@ namespace winrt::TerminalApp::implementation
         CATCH_LOG();
     }
 
+    // Method Description:
+    // - Exports the content of the Terminal Buffer inside the tab
+    // Arguments:
+    // - tab: tab to export
+    winrt::fire_and_forget TerminalPage::_ExportTab(const TerminalTab& tab)
+    {
+        try
+        {
+            if (const auto control{ tab.GetActiveTerminalControl() })
+            {
+                const FileSavePicker savePicker;
+                savePicker.as<IInitializeWithWindow>()->Initialize(*_hostingHwnd);

```

Comment:
Okay so just to be sure, this will work when the Terminal is elevated (run as admin), right? I want to make sure we don't run into the same issue that forced #9760

Reply:
I do believe that it will crash.

Reply:
It is _pickers, of all sorts, through the W.S API_ that is broken, not any specific individual use. We may try/catch this one, but... the best we can do is simply not pop up a dialog when Admin.

Reply:
> It is _pickers, of all sorts, through the W.S API_ that is broken, not any specific individual use. We may try/catch this one, but... the best we can do is simply not pop up a dialog when Admin.

Frick. I'll stick that on #9700